src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -170,6 +176,18 @@ function MoneyRequestView({
 
     let amountDescription = `${translate('iou.amount')}`;
 
+    const hasRoute = TransactionUtils.hasRoute(transaction);
+    const rateID = transaction?.comment.customUnit?.customUnitRateID ?? '0';
+
+    const currency = policy ? policy.outputCurrency : PolicyUtils.getPersonalPolicy()?.outputCurrency ?? CONST.CURRENCY.USD;
+
+    const mileageRate = TransactionUtils.isCustomUnitRateIDForP2P(transaction) ? DistanceRequestUtils.getRateForP2P(currency) : rates[rateID as string];
+    const {unit, rate} = mileageRate;
```

Comment:
does it happen after the newest changes? I cannot recreate any of these issues myself

Reply:
I'm not really fond of adding the hardcoded value. It happens probably because we don't have the rates or rateID fallbacks to '0'. I cannot reproduce it

Reply:
@getusha is it for the same workspace as in the video above?

Reply:
@paultsimura ok thanks! I see it now. Fixed